index.md

```diff
@@ -7,3 +7,13 @@ title: "Hello World"
 # Hello World
 
 This is a test, this is only a test
+
+# Are you a human or are you a replicant?
+## You’re in a desert walking along in the sand when all of the sudden you look down, and you see a tortoise, it’s crawling toward you. You reach down, you flip the tortoise over on its back. The tortoise lays on its back, its belly baking in the hot sun, beating its legs trying to turn itself over, but it can’t, not without your help. But you’re not helping. Why is that?
+1. What do you mean, I'm not helping?
+2. What is a tortoise?
+3. I don't know why I would flip the turtle over in the first place
+
+*what do you mean, I'm not helping?*
+
+*how can she not know what she is*
```

Comment:
What does this mean?

Reply:
If you had watched Blade Runner, you would know what this means.